gossip/src/contact_info.rs

```diff
@@ -647,7 +647,7 @@ pub(crate) fn sanitize_quic_offset(
 }
 
 // Returns the socket at QUIC_PORT_OFFSET from the given one.
-pub(crate) fn get_quic_socket(socket: &SocketAddr) -> Result<SocketAddr, Error> {
+fn get_quic_socket(socket: &SocketAddr) -> Result<SocketAddr, Error> {
```

Comment:
This function still uses QUIC_PORT_OFFSET, if it is to be removed, should this not be gone too?

Reply:
yeah, there will be more follow up patches.
This change is mostly patching `LegacyContactInfo`.
I need to make more changes to fully remove these from `ContactInfo`.